src/systemd_api/server.rs

```diff
@@ -13,66 +13,65 @@ impl SystemdService {
     }
 }
 
-type Stream<T> =
-    Pin<Box<dyn tokio_stream::Stream<Item = std::result::Result<T, Status>> + Send + 'static>>;
+type Stream<T> = Pin<Box<dyn tokio_stream::Stream<Item = Result<T, Status>> + Send + 'static>>;
 
 #[tonic::async_trait]
 impl pb::systemd::unit_control_service_server::UnitControlService for SystemdService {
     async fn get_unit_status(
         &self,
-        _request: tonic::Request<pb::systemd::UnitRequest>,
-    ) -> std::result::Result<tonic::Response<pb::systemd::UnitStatusResponse>, tonic::Status> {
+        _request: Request<pb::systemd::UnitRequest>,
+    ) -> Result<Response<pb::systemd::UnitStatusResponse>, Status> {
```

Comment:
This is already much nicer; could also take it further by using `tonic::Result` to omit the `Status`, or even define our own `type Result<T> = tonic::Result<Response<T>>`.